pgtoolkit/conf.py

```diff
@@ -273,7 +273,8 @@ def __init__(
         raw_line: Optional[str] = None,
     ) -> None:
         self._name = name
-        if isinstance(value, str):
+        # We parse value only if not already parsed from a file
+        if raw_line is None and isinstance(value, str):
```

Comment:
@pgiraud, there's already an `if raw_line is None:` below, can you merge both statements so as to ease readability?

Also, it'd be nice to have a unit test solely for this method instead of having it indirectly covered.